src/frontends/pytorch/src/op/sum.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_sum(NodeContext& context) {
+    bool keep_dims = false;
+    ov::Output<ov::Node> axes;
+    Output<Node> cast;
+    FRONT_END_OP_CONVERSION_CHECK(context.get_input_size() >= 1, "Operation has no inputs.");
```

Comment:
The check for `input_is_none` is sufficient, no need to check input size > 0

Reply:
updated